string limb
geometry_msgs/PoseStamped initPosition
geometry_msgs/PoseStamped endPosition
---
int64 success_bool